Telemetry/CorePing.swift

```diff
@@ -57,6 +61,12 @@ class CorePing: TelemetryPing {
 
         let locale = NSBundle.mainBundle().preferredLocalizations.first!.stringByReplacingOccurrencesOfString("_", withString: "-")
 
+        var usage = profile.prefs.arrayForKey(PrefKeyUsage) as? [Int] ?? []
+        if let time = foregroundTime {
+            usage.append(time)
```

Comment:
This could grow to take up any amount of space if we're never successful at uploading. This could be important in the case that the telemetry servers get shut down or something.